app/posts/find-teacher-training/2021-08-16-standardised-entry-requirement-descriptions.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Standardising entry requirement content
+description: Why we have made the descriptions of entry requirements more consistent
+date: 2021-08-16
+---
+
+Prior to the 2022 to 2023 cycle, providers were asked to describe the academic qualification entry requirements for each course as a single free text field.
+
+As there is a minimum initial teacher training entry requirement across all courses, this led to the standard entry requirements being described in different ways by different providers.
+
+For instance:
+
+> GCSE Maths, English and Science at Grade C or higher or equivalent
+
+> GCSEs: At least a Grade C or Grade 4 at GCSE (or equivalent) in English Language, Mathematics and Science is essential for all applicants.
+
+> GCSE in English (or approved equivalent) at grade C/Level 4 or above GCSE in Mathematics (or approved equivalent) at grade C/Level 4 or above GCSE in Science (or approved equivalent) at grade C/Level 4 or above
+
+These minor inconsistencies may be confusing for candidates, and the requirements weren’t always written in the easiest to understand way.
+
+Instead, for the 2022 to 2023 cycle, we have changed the Publish interface to only ask providers to specify the entry requirements that may differ across courses, namely the [acceptance of pending GCSEs and GCSE equivalency tests](/publish-teacher-training-courses/pending-gcses-equivalency-tests/) and the [minimum degree class required](/publish-teacher-training-courses/degree-entry-requirements/).
```

Comment:
Suggestion: break up the sentence and simplify the language a bit, as below.

'For the 2022 to 2023 cycle, we've changed the Publish interface to only ask providers to specify the entry requirements that may differ across courses. 

These include: 

- whether or not they accept [pending GCSEs and GCSE equivalency tests](/publish-teacher-training-courses/pending-gcses-equivalency-tests/) 
- the [minimum degree class required](/publish-teacher-training-courses/degree-entry-requirements/)'

Reply:
Good idea – I've done this. Although I changed "These include" to "These are" and included an extra bullet, as we may as well list all the changes.